MariaDB.xs

```diff
@@ -619,8 +619,7 @@ dbd_mariadb_get_info(dbh, sql_info_type)
 	    break;
 	}
 	case SQL_MAXIMUM_TABLES_IN_SELECT:
-	    /* newSViv((sizeof(int) > 32) ? sizeof(int)-1 : 31 ); in general? */
-	    retsv= newSViv((sizeof(int) == 64 ) ? 63 : 31 );
```

Comment:
This old code looks like some support for 512bit processors 😁 as `sizeof(int)` returns number of bytes (4).